### ISSUE TITLE
Doesn't track initial page load

### ISSUE THREAD

**Issue by theshanergy** [MatteoGabriele/vue-analytics#4]
Tracking works great except for the initial page load, which isn't tracked at all. I believe this is a result of the page method firing before window.ga is defined. As a workaround I've added a call to the page method in a then() method on the loadScript promise checking on success. The downside is that this seems to result in duplicate pushes to ga if used with the router. Is there a better way to approach this?

**Comment by theshanergy** [comment]
Here is my current implementation, for reference:

`Vue.use(VueAnalytics, { router, debug: true })
loadScript('UA-10749408-27').then((val) => {
  if (val.success) {
    Vue.track.page(router.currentRoute.path)
  }
})`

This doesn't actually seem to be producing duplicate pushes at the moment, but I think that simply comes down to which loads first, the router or ga.

**Comment by MatteoGabriele** [comment]
The tracking of the initial page is fixed with the **v1.3.4**
There's still the problem of the script that needs to load, and it's a little bit tricky.
For now a workaround can be this:
```js
loadScript('UA-XXX-AX').then(() => {
  Vue.use(VueAnalytics, { router })
})
```
I'll have a look later, so for now i won't close this issue

**Comment by MatteoGabriele** [comment]
I've published a new release **v1.4.0** 
You can follow the documentaion [here](https://github.com/MatteoGabriele/vue-analytics#auto-tracking-and-google-analytics-script)

Let me know if you have any other questions about.

**Comment by marktheis** [comment]
@MatteoGabriele Hi is it possible to disable tracking of initial page load?
My plan is to do custom tracking, and currently I'm getting double tracked on initial load.

This is with the auto tracking disabled.

**Comment by MatteoGabriele** [comment]
hi @marktheis yes the first load is auto-tracked by default, I'm going to have a look at the code.
which version are you running now?

**Comment by marktheis** [comment]
@MatteoGabriele Latest: 4.0.0

**Comment by MatteoGabriele** [comment]
@marktheis fix available on 4.0.2
first page hit is now only on auto-tracking enabled but it's still possible to disable that when you are using auto-tracking by set `pageviewOnLoad` property to the options

```js
Vue.use(VueAnalytics, {
  autoTracking: {
    pageviewOnLoad: false
  }
})
```

**Comment by marktheis** [comment]
@MatteoGabriele Thanks. It works 💯
